apps/server/src/modules/tool/common/service/validation/rules/parameter-entry-regex-validator.ts

```diff
@@ -0,0 +1,13 @@
+import { ValidationError } from '@shared/common';
+import { CustomParameter, CustomParameterEntry, ToolParameterValueRegexLoggableException } from '../../../domain';
+import { ParameterEntryValidator } from './parameter-entry-validator';
+
+export class ParameterEntryRegexValidator implements ParameterEntryValidator {
+	public validate(entry: CustomParameterEntry, declaration: CustomParameter): ValidationError[] {
```

Comment:
The service is implicit testing all new parameterEntryValidators but maybe we should create a new test file for each validator in a refactoring ticket? 
Then the service test could be less complex